tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md

```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
 This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
 
 #Test Steps:
-1. Deploy a new vCenter in Nimbus with NSX configured  
-2. Deploy VCH Appliance to the new vCenter
-3. Run a variety of docker commands on the VCH appliance
+1. Deploy NSX
+	/mts/git/bin/nimbus-vsmdeploy --nics 1 --vsmBuild ob-5007049 vic-3-nsxv-mgr
```

Comment:
this isn't providing the formatting that I think you want, might want to try back tics.